From 92ddbb36e2a589fe4d1cf27c30b67a1bb12bf9dc Mon Sep 17 00:00:00 2001
From: Martin Wilck <mwilck@suse.com>
Date: Fri, 16 Nov 2018 11:00:44 +0100
Subject: [PATCH] ui5/systemtray: wait only 10s for system tray

On GNOME >= 3.26, it will never show up. This may cause hangs on
logout if users log in and log out quickly again.
---
 ui5/systemtray.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui5/systemtray.py b/ui5/systemtray.py
index a0cdaf1..101807a 100644
--- a/ui5/systemtray.py
+++ b/ui5/systemtray.py
@@ -841,7 +841,7 @@ def run(read_pipe):
     app.setQuitOnLastWindowClosed(False) # If not set, settings dlg closes app
 
     i = 0
-    while i < 60:
+    while i < 10:
         if QSystemTrayIcon.isSystemTrayAvailable():
             break
         time.sleep(1.0)
-- 
2.21.0

